docs/knowledge_base/policy_development.md

```diff
@@ -1,3 +1,33 @@
+# Policy Basics
+
+This is a basic policy using an `issue` rule to block any HIGH/CRITICAL issues.
+
+```rego
+# METADATA
+# title: Limit risk
+# description: |
+#   Block issues based on risk level.
+package policy.v1
+
+import data.phylum.level
+import rego.v1
+
+# METADATA
+# title: risk level cannot exceed medium
+deny contains issue if {
+    some issue in data.issues
+    issue.severity > level.MEDIUM
+}
+```
+
+The `title` and `description` from the initial metadata comments are displayed in the Phylum UI and are highly recommended.
```

Comment:
maybe a nice touch to show the difference between a properly annotated policy, and a policy without  title and description
<img width="597" alt="Screenshot 2024-08-07 at 1 17 38 PM" src="https://github.com/user-attachments/assets/ac368880-b21d-43e9-9964-8f71c78deb98">

Reply:
I skipped it as you are making some changes to that default behavior/display, correct?

Reply:
good point